### DISCUSSION TITLE
Please add a twitch chat popout button

### DISCUSSION THREAD

**Discussion by lastmetroid** [night/betterttv#6380] [Ideas]
Same functionality like this Chrome addon does:

https://chrome.google.com/webstore/detail/twitch-popout-chat/haijgfkhagnfkbebhgkcgdgkhaljkjcg/related

![image](https://github.com/night/betterttv/assets/6721083/933a5969-1410-43f7-96e4-0acf60669201)

This will work around the issue, that in a normal Chrome window, the width is limited to 500px. This way you can have a way smaller compacter chat window.

**Comment by dclstn** [comment]
Thank you for the suggestion! however this addition seems unlikely given it wouldn't work on user-script.

**Comment by night** [comment]
Closing as resolved — this isn't planned. As noted above, a popout button wouldn't work in BTTV's user-script distribution.
